src/index.js

```diff
@@ -200,3 +200,48 @@ Hooks.on('canvasInit', () => {
 });
 
 Hooks.on('renderChatMessage', (_, html) => Item5e.chatListeners(html));
+
+Hooks.on("createToken", (newNpc) => {
```

Comment:
Since this requires rolling then it should be async `Hooks.call('createToken', async (token) => {`

Reply:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/33249712/215588981-da43eb8c-d34b-4305-b7a3-60b394ca2b25.png">

Yeeeeep